jupyter_book/cli/main.py

```diff
@@ -455,17 +459,23 @@ def sphinx(ctx, path_source, config, toc):
     path_config, full_path_source, config_overrides = ctx.invoke(
         build, path_source=path_source, config=config, toc=toc, get_config_only=True
     )
-
-    sphinx_config, config_meta = get_final_config(
+    sphinx_config, _ = get_final_config(
         user_yaml=Path(path_config) if path_config else None,
         sourcedir=Path(full_path_source),
         cli_config=config_overrides,
     )
-    lines = []
+    lines = ["# auto-generated by jupyter-book"]
```

Comment:
```suggestion
    lines = [
      "###############################################################################",
      "# Auto-generated by `jupyter-book config`",
      "# If you wish to continue using _config.yml, make edits to that file and ",
      "# re-generate this one.",
      "###############################################################################"
    ]
```

Reply:
I think this could make it less likely that people accidentally make edits to the wrong file. If we don't want a message this big perhaps we could just keep your message but add more characters to draw the attention more? I find it confusing when auto-generated things don't make it really obvious that they're auto-generated

Reply:
sounds good 👍